addons/website_sale/static/src/snippets/s_dynamic_category_snippet/000.scss

```
@@ -0,0 +1,30 @@
+.s_category_container{
+    grid-auto-rows: minmax(15vh, auto);
+}
```

```suggestion
.s_category_container {
    grid-auto-rows: minmax(15vh, auto);
    
    > div {
    overflow: hidden;
  }
}
```
Better to use scss formatting